src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -388,16 +504,14 @@ export class QuickBar extends LitElement {
           }
         }
 
-        ha-icon {
-          margin-left: 8px;
-        }
-
+        ha-icon,
         ha-svg-icon {
-          color: var(--primary-text-color);
+          margin-left: 8px;
         }
 
         ha-svg-icon.prefix {
           margin: 8px;
+          color: var(--primary-text-color);
```

Comment:
Should only use primary text color for the prefix icon. The rest should stay default